data/configurables/plane_designs.txt

```diff
@@ -0,0 +1,288 @@
+plane_design = {
```

Comment:
Adding some obsolete designs wouldn't be bad, for minors that want cheaper planes or something, maybe?
Are there any such obsolete designs in base hoi4?